sycl/test/basic_tests/esimd/slm_load4.cpp

```diff
@@ -0,0 +1,23 @@
+// RUN: %clangxx -fsycl -fsycl-explicit-simd -fsycl-device-only -fsyntax-only -Xclang -verify %s
+// expected-no-diagnostics
```

Comment:
Can we have tests with real execution rather than just frontend checks?

Reply:
We would have functional tests once we upstream the whole ESIMD extension.